src/ucp/core/ucp_am.c

```diff
@@ -661,16 +661,15 @@ static ucs_status_t ucp_am_zcopy_multi(uct_pending_req_t *self)
 
     if (req->send.state.dt.offset != 0) {
         ucp_am_fill_middle_header(&mid_hdr, req);
-        return ucp_do_am_zcopy_multi(self, UCP_AM_ID_FIRST, UCP_AM_ID_MIDDLE,
+        return ucp_do_am_zcopy_multi(self, UCP_AM_ID_LAST, UCP_AM_ID_MIDDLE,
```

Comment:
Why need to change?

Reply:
we don't provide the HDR, so better to not provide an AM ID too

Reply:
seems unrelated to this PR: in this PR, need to only change eager_sync

Reply:
removed